docker/default_image/Dockerfile.gpu

```diff
@@ -0,0 +1,124 @@
+FROM nvidia/cuda:9.2-runtime
```

Comment:
I searched around the internet for a bit to figure this out but there doesn't seem to be a clean way because they are derived from different base images. 

The only way to do this would be to do some command-line shenanigans ourselves to dynamically insert the shared commands into a temporary Dockerfile when building the images, but I don't know if it is worth it.

Reply:
We can have both docker files have a block that is meant to be identical and put a comment to that effect.

Reply:
That was my intention with the `# CPU specific commands below` `# GPU specific commands below` comments but can make that more explicit, especially after getting rid of the huge comment block on top.